<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSLProperties</class>
 <widget class="QDialog" name="LSLProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRecord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>170</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>record</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxStream">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>170</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>123</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>id name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
